crates/telio-model/src/constants.rs

```diff
@@ -1,8 +1,17 @@
 //! Telio constants definition
 
+use std::net::Ipv4Addr;
+
 /// VPN IPv4 Meshnet Address
 pub const VPN_INTERNAL_IPV4: [u8; 4] = [100, 64, 0, 1];
 /// VPN IPv6 Meshnet Address
 pub const VPN_INTERNAL_IPV6: [u16; 8] = [0xfd74, 0x656c, 0x696f, 0, 0, 0, 0, 1];
 /// VPN IPv4 Non-Meshnet Address
 pub const VPN_EXTERNAL_IPV4: [u8; 4] = [10, 5, 0, 1];
+
+/// STUN IPv4 Meshnet Address
+pub const STUN_INTERNAL_IPV4: Ipv4Addr = Ipv4Addr::new(100, 64, 0, 4);
+/// Starcast Virtual Peer IPv4 Meshnet Address
+pub const STARCAST_VPEER_INTERNAL_IPV4: Ipv4Addr = Ipv4Addr::new(100, 64, 0, 5);
+/// Starcast Virtual Peer Multicast Address
+pub const STARCAST_VPEER_MULTICAST_IPV4: Ipv4Addr = Ipv4Addr::new(224, 0, 0, 4);
```

Comment:
so it should be a ip network 224.0.0.0/4 not ip 224.0.0.4

Reply:
Also, this multicast ip network will be more dynamic, so I think it's simplest to simply remote it and check just by a reserved ip.

Reply:
This should only be used to check if a peer is a multicast peer, so checking within that network range should be enough. I'll make the change